aws-scala/src/main/scala/Main.scala

```diff
@@ -0,0 +1,13 @@
+package myproject
+
+import com.pulumi.{Context, Pulumi}
+import com.pulumi.aws.s3.Bucket
+
+object App {
+  def main(args: Array[String]): Unit = {
+    Pulumi.run { (ctx: Context) =>
+      var bucket = new Bucket("my-bucket");
+      ctx.`export`("bucketName", bucket.bucket())
+    }
```

Comment:
Most of our templates have comments that call out the important bits, so would be good to add them here, too:

```suggestion
    Pulumi.run { (ctx: Context) =>

      // Create an AWS resource (S3 Bucket)
      var bucket = new Bucket("my-bucket");

      // Export the name of the bucket
      ctx.`export`("bucketName", bucket.bucket())
    }
```